### ISSUE TITLE
Loaders are unavailable within JSS

### ISSUE THREAD

**Issue by markdalgleish** [markdalgleish/jss-loader#2]
The most obvious example of this problem is that compilation fails if your JSS depends on another JSS file (e.g. when using CSS Modules' `composes`).

This is a critical issue that is likely to cause this project to become deprecated in favour of a Webpack plugin, along the lines of @andreypopp's [Styling](https://github.com/andreypopp/styling).

**Comment by geekyme** [comment]
I don't think we are also able to import vendor stylesheets like bootstrap from within a JSS right? That seems like a deal breaker.

**Comment by kof** [comment]
@markdalgleish not sure what you mean .... maybe an example can help ....

**Comment by joshgillies** [comment]
Correct me if I'm wrong but I think what @markdalgleish is referring to is currently the loader lacks the ability to pass compose statements back into Webpacks loader pipeline.

As an example:

colour.jss.js

``` js
module.exports = {
  '.blue': {
    color: 'blue'
  },
  '.green': {
    color: 'green'
  }
};
```

component.jss.js

``` js
module.exports = {
  '.hello': {
    composes: 'blue from "./colour.jss.js"'
  },
  '.world': {
    composes: 'green from "./colour.jss.js"'
  }
};
```

component.js

``` js
var styles = require('css?modules!jss!./component.jss.js');

var html = '<div class="' + styles.hello + '">Hello World!</div>';
```

**Comment by joshgillies** [comment]
Hrmm, actually the above statement isn't correct. `composes` seems to work exactly as expected in the above case. @markdalgleish when you get a moment care to chime in with regards to this issue?
